### PR TITLE
release AuthenticationService

### PR DESCRIPTION
**Pull request by u3IOm4uk**

### CODE REVIEW

File: src/main/java/mate/academy/Main.java [hunk 1]
```diff
@@ -21,4 +21,5 @@ private static void test(String email, String password, boolean expected) {
             System.out.println("Email: " + email + ", password " + password);
         }
     }
+
```
Comment (okuzan) [inline src/main/java/mate/academy/Main.java:24]:
```suggestion
```

File: src/main/java/mate/academy/model/User.java [hunk 2]
```diff
@@ -16,4 +16,5 @@ public String getEmail() {
     public String getPassword() {
         return password;
     }
+
```
Comment (okuzan) [inline src/main/java/mate/academy/model/User.java:19]:
```suggestion
```

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 3]
```diff
@@ -10,7 +12,13 @@ public class AuthenticationService {
...
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user != null) {
+            return password.equals(user.getPassword());
+        }
         return false;
```
Comment (okuzan) [inline src/main/java/mate/academy/service/AuthenticationService.java:22]: make UserService a private class field
return boolean expression , without using true/false literals

**Review by okuzan — CHANGES_REQUESTED** [review]

**Review by okuzan — CHANGES_REQUESTED** [review]
ignore this comment

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 4]
```diff
@@ -10,7 +12,11 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+        return password.equals(user.getPassword());
```
Comment (Ivan95kos) [inline src/main/java/mate/academy/service/AuthenticationService.java:19]: You can get the NullPointerException,
Additional verification is required
```suggestion
        return user != null && user.getPassword().equals(password);
```

**Review by Ivan95kos — APPROVED** [review]
Great job, with minor comments

**Pull request closed without merging**